ndicapi.cxx

```diff
@@ -68,7 +68,7 @@ namespace
     // call the user-supplied callback function
     if (pol->ErrorCallback)
     {
-      pol->ErrorCallback(errnum, ndiErrorString(errnum), pol->ErrorCallbackData);
+      pol->ErrorCallback(errnum, (char*)ndiErrorString(errnum), pol->ErrorCallbackData);
```

Comment:
So, the return type of ndiErrorString is `char*`, so this is redundant. Is it the `ndicapiExport` token in `ndicapiExport char* ndiErrorString(int errnum);` that causes the warning?

Reply:
Oh, I see, changes to ndiErrorString below, nevermind.